ostd/src/mm/page/mod.rs

```diff
@@ -363,3 +330,22 @@ impl Drop for DynPage {
         }
     }
 }
+
+/// Increase the reference count of the page by one.
+///
+/// # Safety
+///
+/// The physical address must represent a valid page.
+///
+/// And the caller must ensure the metadata slot pointed through the corresponding
+/// virtual address is initialized by holding a reference count of the page firstly.
+/// Otherwise the function may add a reference count to an unused page.
+pub(in crate::mm) unsafe fn inc_page_ref_count(paddr: Paddr) {
+    debug_assert!(paddr % PAGE_SIZE == 0);
+    debug_assert!(paddr < MAX_PADDR.load(Ordering::Relaxed) as Paddr);
+    let vaddr: Vaddr = mapping::page_to_meta::<PagingConsts>(paddr);
+    // SAFETY: The virtual address points to an initialized metadata slot.
+    (*(vaddr as *const MetaSlot))
```

Comment:
Try to add an `unsafe` block here as we have safety comments now. Someone trying to enable `unsafe_op_in_unsafe_fn` lint will benefit from this.